bin/mustache

```diff
@@ -131,6 +137,27 @@ function hasVersionArg () {
   });
 }
 
-function getPartialName (filename) {
-  return path.basename(filename, '.mustache');
+function getPartialNames (filename) {
+
```

Comment:
Repo's code style is not to have a blank line in the beginning of a function. It also feels inconsistent if the function doesn't end with a blank line as well, but that might be my opinion. (same for the `onDone` function)